File: app/src/main/java/com/booboot/vndbandroid/model/vndb/Screen.java
package com.booboot.vndbandroid.model.vndb;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

/**
 * Created by od on 12/03/2016.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class Screen {
    private String image;
    private int rid;
    private boolean nsfw;
    private int height;
    private int width;

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public int getRid() {
        return rid;
    }

    public void setRid(int rid) {
        this.rid = rid;
    }

    public boolean isNsfw() {
        return nsfw;
    }

    public void setNsfw(boolean nsfw) {
        this.nsfw = nsfw;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }
}

File: app/src/main/java/com/booboot/vndbandroid/model/vndb/Screen.kt
package com.booboot.vndbandroid.model.vndb

import com.squareup.moshi.JsonClass
import java.io.Serializable

@JsonClass(generateAdapter = true)
data class Screen(
    var image: String = "",
    var rid: Int = 0,
    var nsfw: Boolean = false,
    var height: Int = 0,
    var width: Int = 0
) : Serializable